### ISSUE TITLE
Windows failing tests

### ISSUE THREAD

**Issue by XhmikosR** [shelljs/shelljs#127]
```
> shelljs@0.3.0 test C:\Users\xmr\Desktop\shelljs
> node scripts/run-tests

Running test: cat.js
Running test: cd.js
Running test: chmod.js

assert.js:92
  throw new assert.AssertionError({
        ^
AssertionError: 438 == 493
    at Object.<anonymous> (C:\Users\xmr\Desktop\shelljs\test\chmod.js:23:8)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:906:3
*** TEST FAILED! (missing exit code "123")

Running test: common.js
Running test: config.js
Running test: cp.js

assert.js:92
  throw new assert.AssertionError({
        ^
AssertionError: 0 == 1
    at Object.<anonymous> (C:\Users\xmr\Desktop\shelljs\test\cp.js:142:8)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:906:3
*** TEST FAILED! (missing exit code "123")

Running test: dirs.js
Running test: echo.js
Running test: env.js
Running test: exec.js
Running test: find.js
Running test: grep.js
Running test: ln.js

assert.js:92
  throw new assert.AssertionError({
        ^
AssertionError: false == true
    at Object.<anonymous> (C:\Users\xmr\Desktop\shelljs\test\ln.js:51:1)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:906:3
*** TEST FAILED! (missing exit code "123")

Running test: ls.js
Running test: make.js
Running test: mkdir.js
Running test: mv.js
Running test: popd.js
Running test: pushd.js

assert.js:92
  throw new assert.AssertionError({
        ^
AssertionError: "resources\\pushd" == "resources/pushd"
    at Object.<anonymous> (C:\Users\xmr\Desktop\shelljs\test\pushd.js:193:8)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:906:3
*** TEST FAILED! (missing exit code "123")

Running test: pwd.js
Running test: rm.js
Running test: sed.js
Running test: tempdir.js
Running test: test.js

assert.js:92
  throw new assert.AssertionError({
        ^
AssertionError: false == true
    at Object.<anonymous> (C:\Users\xmr\Desktop\shelljs\test\test.js:76:8)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:906:3
*** TEST FAILED! (missing exit code "123")

Running test: to.js
Running test: toEnd.js
Running test: which.js

*******************************************************
WARNING: Some tests did not pass!
*******************************************************
npm ERR! Test failed.  See above for more details.
npm ERR! not ok code 0

```

**Comment by cvrebert** [comment]
Duplicate of #296?

**Comment by nfischer** [comment]
Fixed in #297
